staging/src/k8s.io/kube-aggregator/pkg/apiserver/openapi_aggregator.go

```diff
@@ -110,6 +138,10 @@ type byPriority struct {
 func (a byPriority) Len() int      { return len(a.specs) }
 func (a byPriority) Swap(i, j int) { a.specs[i], a.specs[j] = a.specs[j], a.specs[i] }
 func (a byPriority) Less(i, j int) bool {
+	// All local specs will come first
+	if a.specs[i].apiService.Spec.Service == nil {
```

Comment:
Building something inconsistent here doesn't feel good. IMO we have to follow the architecture of the API aggregation much more than working around it and creating an approximation of the logic on the API side. Not that I have the perfect solution here though either :-/